### ISSUE TITLE
Very few screenshots used in UG

### ISSUE THREAD

**Issue by taniatsq** [taniatsq/pe#10]
There were only 2 screenshots used in the user guide (one of the initial ui set up, and another on the help feature).

None of the main features had any pictures for readers to look and follow through. As such, commands with many constraints (like `find`) and commands that aren't so intuitive especially those concerning indexes (like `addfav`, `addorder`) might not be very clear to readers using the app for the first time.

![Screenshot 2024-04-19 at 5.17.51 PM.png](https://raw.githubusercontent.com/taniatsq/pe/main/files/fc4da570-5dd1-432c-8675-e7cca1c0ff03.png)

See that the instructions for `find` in the user guide is almost a page full of words, and no pictures.

**Comment by nus-pe-script** [comment]
[IMPORTANT!: Please do not edit or reply to this comment using the GitHub UI. You can respond to it using CATcher during the next phase of the PE]

# Team's Response
No details provided by team.

## The 'Original' Bug

[The team marked this bug as a duplicate of the following bug]

># Insufficient screenshots in the UG
>
>**Note from the teaching team:** This bug was reported during the _Part II (Evaluating Documents)_ stage of the PE. **You may reject this bug if it is not related to the quality of documentation.**<hr>
>There is only one screenshot found of the application. There are no screenshots to support the command explanations. Hence, it appears too wordy.
>
>
>![image.png](https://raw.githubusercontent.com/nus-cs2103-AY2324S2/pe/master/files/5abcd462-a64d-4a78-96a6-5e772b7e48a1.png)
>
>
>
>
>
><hr>
><sub>[original: nus-cs2103-AY2324S2/pe-interim#4070] [original labels: severity.Medium type.DocumentationBug]</sub>
>
>
>

## Their Response to the 'Original' Bug

[This is the team's response to the above 'original' bug]

>Mostly cosmetic / UI issue but this is a good catch nonetheless.

# Items for the Tester to Verify

## :question: Issue duplicate status

Team chose to mark this issue as a duplicate of another issue (as explained in the _**Team's response**_ above)

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

<catcher-end-of-segment><hr>
